proto/proxy/accept_policy/accept_policy.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+package conduit.proxy.accept_policy;
+
+import "common/common.proto";
+
+service AcceptPolicy {
+  // Describes how a proxy should accept inbound connections from remote peers.
+  //
+  // Includes;
+  // - Protocol detection
+  // - [TODO] TLS termination
+  rpc Inbound(InboundAcceptPolicyRequest) returns (stream InboundAcceptPolicy) {}
+
+  // Describes how a proxy should accept outbound connections from local peers.
+  //
+  // Includes;
+  // - Protocol detection
+  rpc Outbound(OutboundAcceptPolicyRequest) returns (stream OutboundAcceptPolicy) {}
+}
+
+message InstanceId {
+  // TODO
```

Comment:
Is this planned to be a unique opaque identifier?  I think this probably needs to include enough information that the controller can recover information about the pod (e.g. pod name, or pod owner name), even if this is opaque to the proxy.  Presumably we will want to be able to set different AcceptPolicy for different deployments, etc.